packages/client/hmi-client/src/App.vue

```diff
@@ -9,10 +9,12 @@
 		:active="!isErrorState"
 		:current-project-id="project?.id ?? null"
 		:projects="projects"
-		:resourceType="resourceType"
+		:resourceType="pageRef?.resourceType ?? ResourceType.XDD"
```

Comment:
I'll prefer that you make this a computed, and that you explain where `resourceType` is coming from with the `defineExpose` function in `DataExplorer.vue`.
I'm not even sure I like this patern; but for now we need to be explicit as to not search for it.

Reply:
I decided to take this a bit further by passing `showSuggestions` which is the only thing the children need from `resourceType`